src/current/v22.2/backup-validation.md

```diff
@@ -126,7 +126,7 @@ It is necessary to include `schema_only` when you run a restore with `verify_bac
 RESTORE DATABASE movr FROM LATEST IN "s3://bucket?AWS_ACCESS_KEY_ID={Access Key ID}&AWS_SECRET_ACCESS_KEY={Secret Access Key}" WITH schema_only, verify_backup_table_data;
 ~~~
 
-Similarly, to just `schema_only` restores, you'll find the table schemas restored. If a file is not present or unreadable in the backup, you'll receive an error. 
+Similarly, to just `schema_only` restores, you'll find the table schemas restored. If a file is not present or unreadable in the backup, you'll receive an error.
```

Comment:
```suggestion
Like `schema_only` restores, restoring with `verify_backup_table_data`  restores table schemas. If a file is not present or unreadable in the backup, you'll receive an error.
```

I think this belongs before line 120, to present similarities before differences. It doesn't make much sense to me to restore table data without restoring the schema first.

Reply:
Yes, I think that's a good argument, moved.